### PR TITLE
Add linux aarch64 wheel build support

### PR DESCRIPTION
**Pull request by odidev**
Add linux aarch64 wheel build support. 

Related to https://github.com/noirello/pyorc/issues/42 @noirello Could you please review this PR?

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -170,6 +170,51 @@ stages:
...
         pathtoPublish: '$(Build.ArtifactStagingDirectory)'
         artifactName: wheels
+  - job: manylinux_aarch64_wheels
+    pool:
+      vmImage: "ubuntu-latest"
+    strategy:
+      matrix:
+        Python36:
+          cpython.version: cp36-cp36m
+          manylinux: 'manylinux_aarch64'
```
Comment (noirello) [inline azure-pipelines.yml:180]: Are these `manylinux` attributes necessary in the matrix? I can't see any reference in the build later on.

Comment (odidev) [reply azure-pipelines.yml]: yes, it is not required. Changes are done.

**Review by noirello — APPROVED** [review]
LGTM, thank you for the contribution.
Did you have the chance to test the artifacts on a native aarch64 system?

**Comment by odidev** [comment]
> Did you have the chance to test the artifacts on a native aarch64 system?

Yes, I have tried to test the wheel on aarch64 but 4 test cases are failing on the AArch64 architecture. Please refer the logs for the error.

**Log file:** 
[pyorc_outputfile.txt](https://github.com/noirello/pyorc/files/7194148/pyorc_outputfile.txt)

**Comment by noirello** [comment]
Maybe it related to endianness. Could you please check the result of `sys.byteorder`? Does it return `big` on your system?

**Comment by odidev** [comment]
@noirello The result of `sys.byteorder`:

```
>>> import sys
>>> sys.byteorder
'little'
>>>
```

**Pull request merged**